src/tasks/birthday.ts

```diff
@@ -0,0 +1,78 @@
+import { getHandler } from '#languages';
+import { PartialResponseValue, ResponseType, Task } from '#lib/database';
+import { TextChannel, User } from 'discord.js';
+
+const enum Matches {
+	Age = '{age}',
+	AgeOrdinal = '{ageOrdinal}',
+	User = '{user}'
```

Comment:
This isn't enough featureful and doesn't have consistency [with tags](https://gist.github.com/kyranet/485287383d818ef70fe347e45f4f63a5).

`{user}` should mention, `{user.name}` should use the username, `{user.tag}` should show the tag, and so on.

Also, `{age.ordinal}` would look more consistent with the others this way.